### ISSUE TITLE
Missing customer_uid attribute

### ISSUE THREAD

**Issue by rroupski** [ocsf/ocsf-schema#157]
The customer_uid attribute is missing from the base event. We need it to handle multi-tenant systems such as Splunk on cloud that handles events from multiple customers.

**Comment by pagbabian-splunk** [comment]
I believe customer_uid was a metadata attribute rather than a base event attribute.

**Comment by floydtree** [comment]
Agreed, it's a better suited field for metadata. We had discussed it [here](https://github.com/ocsf/ocsf-schema/pull/135) as well.

Alternative is to create implementation specific profiles and add such fields to these profiles. Selecting a profile would overlay such fields for every class.
